src/hotspot/share/gc/shared/genCollectedHeap.hpp

```diff
@@ -175,6 +177,10 @@ class GenCollectedHeap : public CollectedHeap {
 
   size_t capacity() const;
   size_t used() const;
+  size_t live() const {
+    size_t live = _live_size;
+    return live > 0 ? live : used();
+  };
```

Comment:
I think the implementation belongs to `genCollectedHeap.cpp`.

Reply:
+1. Does not seem to be performance sensitive.

Reply:
👍